base/lnls-get-n-unpack.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+#
+# Download and extract tarball archive from the network.
+
+set -eu
+
+case "$1" in
+    -r) dest=/ ;;
+    -l) dest=. ;;
+    *) >&2 echo "Invalid extraction mode: must be either root (-r) or local (-l)."
+       exit 1;
+    ;;
+esac
+
+shift
+
+for url; do
+    download_dir=$(mktemp -d)
+
+    echo Downloading $(basename "$url")...
+    wget -P $download_dir -o $download_dir/wget.log "$url" || (cat $download_dir/wget.log && false)
+    tar --no-same-owner -xf $download_dir/*.tar* -C $dest
```

Comment:
Couldn't we use `$download_dir/*` here? This doesn't support stuff like `.tgz` files.

Reply:
We can if we log `wget` elsewhere or remove the log file afterwards. It might be a better to do so than to restrict the filename indeed. I'll update it. :)